src/mjml-button.tsx

```diff
@@ -9,7 +10,16 @@ export const MjmlButton = ({
 }: React.PropsWithChildren<
   MjmlButtonProps & PaddingProps & ClassNameProps & HrefProps & BorderProps
 >) => {
-  return React.createElement("mj-button", handleMjmlProps(rest), children);
+  return renderTo({
+    html: React.createElement("mj-button", handleMjmlProps(rest), children),
+    text: children ? (
```

Comment:
Wdyt of calling this `mjml / simpleHtml` rather than `html / text`?

Reply:
I started to like textish HTML, reminds me of [bestish mensch](https://youtu.be/_BihD5SzOvg) haha

`mjml / simpleHtml` sound good to me, though `simpleHtml` lacks a bit of a pointer to what this is going to be used for, that is converting to text (which eg. explain why we wouldn't use any paddings, colors and everything is a div or a line break haha)

What do you think of `mjml / textHtml`? Or `htmlForText`? Doesn't make too big of a difference at the end of the day…

Replacing to `mjml / simpleHtml`, let me know if any of other suggestions sounds better to you!

Reply:
Yea your call!